src/backend/database_models/agent.py

```diff
@@ -18,6 +18,7 @@ class Agent(Base):
 
     tools: Mapped[list[str]] = mapped_column(JSON, default=[], nullable=False)
     tools_metadata: Mapped[list[AgentToolMetadata]] = relationship("AgentToolMetadata")
+    is_private: Mapped[bool] = mapped_column(Boolean, default=False, nullable=True)
```

Comment:
No need to make it nullable=True since you set a default

Reply:
Wouldn't it break for existing rows in the DB tho? Or do I need either a default or nullable?